JavaScript/call-apply-bind.md

```diff
@@ -0,0 +1,148 @@
+# Call(), Apply() & Bind()
+___
+
+#### In essence, the purpose of the Call(), Apply(), Bind() methods is to enable the ability to control what the ```this``` variable ends up being when a function is run.
+___
+
+##  What are call(), apply() & bind() methods?
+
+```call```, ```apply``` & ```bind``` are simply methods which are created - by default - by the JavaScript engine as properties on every function object. Exactly the same way you can have methods as properties of a standard JavaScript object.  This is made possible because of the fact that technically, JavaScript functions are classed as objects.
```

Comment:
+```call```, ```apply``` & ```bind``` are simply methods which are created - by default - by the JavaScript engine as properties on every function object. Exactly the same way you can have methods as properties of a standard JavaScript object.  This is made possible because of the fact that technically, JavaScript functions are classed as objects.  
  

In exactly the same way that methods as properties are of a standard JavaScript object.